core/baseline-core-v1.0-psd01.md

```diff
@@ -2786,113 +2795,722 @@ The document defines storage capability requirements in the following areas:
 * Fully Persistent Data
 
 ## 7.1 BPI Storage Security
+###### BPISTORAGESECURITY
 
 As has been done throughout this document, there are BPI layer-specific security requirements for layers and/or components which are listed below. It is assumed that the BPI Storage capabilities are required to fulfill all BPI security requirements in [2 Design and Architecture](#2-design-and-architecture). 
 
 #### **[R279]** 
 Data in transit in a BPI MUST be encrypted.
 
+[[R279]](#r279) Testability: 
+
+Preconditions: 
+
+* There is a BPI test environment with at least two independent services processing data that communicate with each other
+* The communication between the services is captured using a packet capture system
+* The encryption algorithm(s) in the test meets the security requirements of this document.
+
+Test Steps:
+
+1. Start the communication between the two services in the test environment using some plain text test data which is encrypted/decrypted by each service.
+2. Capture the network traffic using the packet capture tool.
+3. Inspect the captured packets to confirm that all data transmitted between the nodes is encrypted, and not in plain text, and can be decrypted using the key(s) used for encryption.
+5. Repeat steps 1-3 for different types of data and communication scenarios (e.g., different data formats, different network topologies).
+
+Expected Results: All data transmitted between the nodes is encrypted (does not appear in plain text).
+
+The requirement is met, if all expected results are met. The requirement is not me, if any of the results are not meeting test expectations.
+
 #### **[R280]** 
 Data at rest in a BPI MUST be encrypted.
 
+[[R280]](#r280) Testability: 
+
+Preconditions: 
+
+* There is a BPI test environment that can store data
+* The encryption algorithm(s) in the test meets the security requirements of this document.
+
+Test Steps:
+
+1. Identify the location of the data at rest in the BPI.
+2. Verify that the data is encrypted and cannot be decrypted without the encryption/decryption key.
```

Comment:
```suggestion
2. Verify that the data is encrypted with the encryption/decryption key.
```
You can verify data has been encrypted. You can't demonstrate that it is impossible to decrypt - and that isn't the requirement stated.